pytest_pootle/fixtures/models/store.py

```diff
@@ -370,14 +370,6 @@ def ru_tutorial_po(settings, russian_tutorial):
 
 
 @pytest.fixture
-def ru_update_save_changed_units_po(settings, russian_tutorial):
-    """Require the /ru/tutorial/tutorial.po store."""
-    return _require_store(russian_tutorial,
-                          settings.POOTLE_TRANSLATION_DIRECTORY,
-                          'update_save_changed_units.po')
```

Comment:
Just wondering if the files in https://github.com/phlax/pootle/tree/master/pytest_pootle/data/po/tutorial/ru should be dropped too.